pkg/appgw/configbuilder.go

```diff
@@ -214,7 +214,7 @@ func generateListenerID(ingress *v1beta1.Ingress, rule *v1beta1.IngressRule, pro
 	if protocol == n.HTTPS {
 		frontendPort = Port(443)
 	}
-	if overridePort != nil {
+	if overridePort != nil && *overridePort > 0 && *overridePort < 65000 {
```

Comment:
How about having a V5 logging to indicate a warning such as port out of boundary when annotation is enabled?

Reply:
Added the V5 for out of bound configuration